### PR TITLE
[ ZEPPELIN-531 ] execute and stop note in job menu

### PR DESCRIPTION
**Pull request by cloverhearts**
### What is this PR for?
We can note start and stop in job menu.

### What type of PR is it?
Improvement

### Todos
- [x] - add feature for run/stop

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-968

### How should this be tested?
Click to a note start or stop button in job menu.

### Screenshots (if appropriate)
executenote

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by astroshim** [comment]
It's working as expected. Thank you for great improvement.

**Comment by cloverhearts** [comment]
@astroshim 
Thank you, have a good day!

**Comment by bzz** [comment]
@cloverhearts thank you for contribution! In my experience, if PR author or one of reviewers post the reason for CI failure - that speeds up the review\merge process dramatically.

CI is failing with flaky pySpark tests, not relevant to the PR

```
Results :

Failed tests: 
  ZeppelinSparkClusterTest.pySparkDepLoaderTest:430 expected:<FINISHED> but was:<ERROR>
  ZeppelinSparkClusterTest.pySparkAutoConvertOptionTest:312 expected:<FINISHED> but was:<ERROR>

Tests run: 82, Failures: 2, Errors: 0, Skipped: 1
```

It will be handled under [ZEPPELIN-1761](https://issues.apache.org/jira/browse/ZEPPELIN-1761).

\cc @felizbear to review the frontend part.

File: zeppelin-web/src/app/jobmanager/jobs/job.controller.js [hunk 1]
```diff
@@ -37,6 +37,68 @@
...
       return isNaN(result) ? 0 : result;
     };
+
+    $scope.runNotebookJob = function(notebookId) {
+      BootstrapDialog.confirm({
+        closable: true,
+        title: '',
+        message: 'Run all paragraphs?',
+        callback: function(result) {
+          if (result === true) {
```
Comment (felizbear) [inline zeppelin-web/src/app/jobmanager/jobs/job.controller.js]: could you please change it to `if (result)` here and elsewhere?
https://github.com/airbnb/javascript#comparison--shortcuts

Comment (cloverhearts) [inline zeppelin-web/src/app/jobmanager/jobs/job.controller.js]: @felizbear 
Thank you for your review!
I will change it :)

File: zeppelin-web/src/app/jobmanager/jobs/job.controller.js [hunk 2]
```diff
@@ -38,6 +38,68 @@
...
+      });
+    };
+
+    $scope.stopNotebookJob = function(notebookId) {
+      BootstrapDialog.confirm({
+        closable: true,
+        title: '',
+        message: 'Stop all paragraphs?',
+        callback: function(result) {
+          if (result === true) {
```
Comment (felizbear) [inline zeppelin-web/src/app/jobmanager/jobs/job.controller.js]: @cloverhearts thanks for addressing my comment. however, you seemed to ignore the "elsewhere" part of it. could you fix it here as well?

File: zeppelin-web/src/app/jobmanager/jobs/job-control.html [hunk 3]
```diff
@@ -18,14 +18,25 @@
...
     {{getProgress()}}%
   </span>
+  <!-- Run / Cancel button -->
+  <span
+    ng-if="notebookJob.isRunningJob === false"
+    class="icon-control-play" style="cursor:pointer;color:#3071A9" tooltip-placement="left" tooltip="START ALL Job"
+    ng-click="runNotebookJob(notebookJob.noteId)">
+  </span>
+  <span
+    ng-if="notebookJob.isRunningJob === true"
```
Comment (felizbear) [inline zeppelin-web/src/app/jobmanager/jobs/job-control.html]: ...and here

File: zeppelin-web/src/app/jobmanager/jobs/job-control.html [hunk 4]
```diff
@@ -18,14 +18,25 @@
...
+      READY
     </span>
   </span>
 
   <span ng-if="notebookJob.isRunningJob === true">
     {{getProgress()}}%
   </span>
+  <!-- Run / Cancel button -->
+  <span
+    ng-if="notebookJob.isRunningJob === false"
```
Comment (felizbear) [inline zeppelin-web/src/app/jobmanager/jobs/job-control.html]: ...and here

**Comment by bzz** [comment]
@cloverhearts can you please rebase it on latest master - `git rebase master` and force push it? This will trigger CI with latest fixes and help to see if there are any relevant failures here.

**Comment by cloverhearts** [comment]
@bzz Sure!
I will That.

**Comment by cloverhearts** [comment]
Hello @bzz ,
could you please review?
ci is green.
I think ready for merge on this pr.
Thank you :)
Have a nice day.

**Comment by cloverhearts** [comment]
ping

**Comment by Leemoonsoo** [comment]
Tested and LGTM.
Merge to master if there're no further comments.

**Pull request closed without merging**